### PR TITLE
[DEV-2377] Update the supported languages for the different fidelities

### PR DESCRIPTION
**Pull request by BrunoMeyer**
# Contents of the *Pull Request*

- Update the supported languages for the different fidelities (considering source and target)
- Change the table display format for the languages

## Jira Tasks Associated

This PR is associated with the following Jira Tasks:
 
[DEV-2377] Update docs for professional languages

### CODE REVIEW

File: api/source/basics.rst [hunk 1]
```diff
@@ -99,232 +99,419 @@ Languages
...
-| Zulu                            | zu                 |
-+---------------------------------+--------------------+
+Transcription and translation are supported in the following languages for HIGH (PROFESSIONAL), MECHANICAL, IMPORT and STANDARD fidelity jobs (volume may vary contact `support <mailto:support@cielo24.com>`_ for more information):
+
+
+High
+**********************
+
++------------------------------------------+---------------------+---------------------+---------------------+
+| Code                                     | Language            | Supported as source | Supported as target |
```
Comment (muhsen84) [inline api/source/basics.rst:109]: @BrunoMeyer , use the same table header in the old docs: https://cielo24.readthedocs.io/en/latest/basics.html#getting-started:~:text=for%20more%20information)%3A-,Language,IETF%20Language%20Code,-Arabic

File: api/source/basics.rst [hunk 2]
```diff
@@ -99,232 +99,419 @@ Languages
...
+| Zulu                                     | zu                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+
+
+
+Mechanical
+**********************
+
++------------------------------------------+---------------------+---------------------+---------------------+
+| Code                                     | Language            | Supported as source | Supported as target |
```
Comment (muhsen84) [inline api/source/basics.rst:341]: Here as well

File: api/source/basics.rst [hunk 3]
```diff
@@ -99,232 +99,419 @@ Languages
...
+| Zulu                                     | zu                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+
+
+
+Standard
+**********************
+
++------------------------------------------+---------------------+---------------------+---------------------+
+| Code                                     | Language            | Supported as source | Supported as target |
```
Comment (muhsen84) [inline api/source/basics.rst:508]: Here as well, the same old header

File: api/source/basics.rst [hunk 4]
```diff
@@ -99,232 +99,419 @@ Languages
...
+| Vietnamese                               | vn                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Xhosa                                    | xh                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Zulu                                     | zu                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+
+
+
+Standard
```
Comment (muhsen84) [inline api/source/basics.rst:504]: Move this table up to be after the HIGH fidelity table

File: api/source/basics.rst [hunk 5]
```diff
@@ -99,232 +99,419 @@ Languages
...
+| Turkish                                  | tr                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Ukrainian                                | uk                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Vietnamese                               | vn                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+
+
+
+Import
```
Comment (muhsen84) [inline api/source/basics.rst:170]: This is not needed as we don't do any processing here, so we support all languages.

**Review by muhsen84 — CHANGES_REQUESTED** [review]

File: api/source/basics.rst (hunk 3 above)
Comment (BrunoMeyer) [reply api/source/basics.rst]: Done

File: api/source/basics.rst (hunk 4 above)
Comment (BrunoMeyer) [reply api/source/basics.rst]: Done

File: api/source/basics.rst (hunk 2 above)
Comment (BrunoMeyer) [reply api/source/basics.rst]: Done

**Review by muhsen84 — APPROVED** [review]

**Pull request merged**